eng/source-build/source-build.proj

```diff
@@ -28,7 +28,7 @@
     <Message Text="Finished restoring Arcade" Importance="High" />
 
     <MSbuild Projects="$(ArcadeDir)/SourceBuild/AfterSourceBuild.proj"
-             Properties="_NETCORE_ENGINEERING_TELEMETRY=AfterSourceBuild;ArcadeBuildFromSource=true;CurrentRepoSourceBuildArtifactsPackagesDir=$(ProjectRoot)artifacts/nupkgs/"
+             Properties="_NETCORE_ENGINEERING_TELEMETRY=AfterSourceBuild;ArcadeBuildFromSource=true;CurrentRepoSourceBuildArtifactsPackagesDir=$(ProjectRoot)artifacts/nupkgs/;FailOnPrebuiltBaselineError=true"
```

Comment:
Does this negatively affect the product source-build?  When doing a full source-build, we don't want this repo to stop the build if it has prebuilts.

Reply:
You're right - it will affect full source-build. I think we should be able to make this change specific to repo source-build. Will update - thanks.

Reply:
Resolved in https://github.com/NuGet/NuGet.Client/pull/5190/commits/eaaaebcd571a201b0bda7ae3e49767824975471a